locales/en/app.ftl

```diff
@@ -368,19 +368,26 @@ dash-tab-remove-title = Remove my data
 dash-tab-beta = Beta
 
 #removal form strings
+
+dash-remove-intro-1 = Mozilla Privacy Pack actively monitors hundreds of thousands of websites for your personal information and removes it from any website that puts you and your loved ones at risk.
+dash-remove-intro-2 = To start your data removal process, please provide the following information to allow us match your profile on the websites exposing your profile and we’ll start the removal process immediately. You’ll be receiveing an email update in 24 hours.
 dash-remove-form-email-label = Select your email address
 dash-remove-form-name-label = What is your full name?
 dash-remove-form-name = Enter your full name
 dash-remove-form-loc-label = What is your current location?
-dash-remove-form-loc-city = Enter your city
-dash-remove-form-loc-state = Enter your state
-dash-remove-form-loc-country = Enter your country
+dash-remove-form-loc-city = City
```

Comment:
Passing by: these changes require new IDs.
For more details see https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html